### PR TITLE
[trtllm handler] Add checks on input data

### PR DESCRIPTION
**Pull request by ydm-amazon**
## Description ##

Adds checks on input data to fix issue with custom preprocessing

### CODE REVIEW

File: engines/python/setup/djl_python/tensorrt_llm.py [hunk 1]
```diff
@@ -69,12 +69,17 @@ def parse_input(self, inputs):
 
         return input_data, input_size, parameters, errors, batch
 
+def is_broken_input_data(input_data):
+    if len(input_data) > 0 and isinstance(input_data[0], dict) and input_data != [{'inputs': ''}]:
```
Comment (lanking520) [inline engines/python/setup/djl_python/tensorrt_llm.py:73]: why we are checking the last piece?

```
[{'inputs': ''}]
```

Comment (ydm-amazon) [reply engines/python/setup/djl_python/tensorrt_llm.py:73]: See quip docs for details

But long story short for all tokens except for first token, input_data is [{'inputs': ''}] no matter the actual request body

Comment (frankfliu) [reply engines/python/setup/djl_python/tensorrt_llm.py:73]: `[{'inputs': ''}]` is expected, why this treat as broken input data?

Comment (ydm-amazon) [reply engines/python/setup/djl_python/tensorrt_llm.py:73]: Exactly
It is expected so if the input_data is `not` equal to `[{'inputs': ''}]` but still contains a dictionary, then it is broken

**Comment by ydm-amazon** [comment]
Ran gradle formatter to pass checks

**Comment by ydm-amazon** [comment]
Fixing issue at toolkit level. Closing.

**Pull request closed without merging**